test/ui/edit_merhant_details_ui_tests.js

```diff
@@ -0,0 +1,56 @@
+'use strict'
```

Comment:
It would be good to apply the same patterns as with the previous PR (see https://github.com/alphagov/pay-selfservice/pull/535). Ui tests are "merged" in `get_test.js` and `post_test.js`. You can use cheerio to assert on ui elements.
The test to add is that when you submit something, it stays in the session (and that the form contains the session values after a post).

Reply:
Thanks @simad . Addressed in [a113be0]